src/opendr/perception/object_detection_3d/voxel_object_detection_3d/dependencies.ini

```
@@ -1,16 +1,17 @@
 [runtime]
 # 'python' key expects a value using the Python requirements file format
-#  https://pip.pypa.io/en/stable/reference/pip_install/#requirements-file-format 
+#  https://pip.pypa.io/en/stable/reference/pip_install/#requirements-file-format
 
 python=torch==1.9.0
        torchvision==0.10.0
+       protobuf<=3.20.0
        tensorboardX>=2.0
        opencv-python==4.5.1.48
        matplotlib>=2.2.2
        tqdm
+       protobuf==3.11.3
```

protobuf appears twice with different version requirements. I assume <=3.20.0 is correct as it appears in other similar files and 3.11.3 was the old one.
```suggestion
       protobuf<=3.20.0
       tensorboardX>=2.0
       opencv-python==4.5.1.48
       matplotlib>=2.2.2
       tqdm
```